spark8t/cli/service_account_registry.py

```diff
@@ -143,6 +146,8 @@ def main(args: Namespace, logger: Logger):
             + parse_conf_overrides(args.conf)
         )
 
+        print(f"ADD CONFIG: {account_configuration}")
```

Comment:
**question**: Is this something we want to print to stdout? 
**nit**: If yes, to make it consistent with REMOVE_CONFIG case, would it be better to just print the added configurations?

Reply:
No, just a leftover from a debugging session :D